SHTSensor.cpp

```diff
@@ -133,12 +133,12 @@ bool SHTI2cSensor::readSample()
   mTemperature = mA + mB * (val / mC);
 
   val = (data[3] << 8) + data[4];
-  mHumidity = mX * (val / mY);
+  mHumidity = mZ + mX * (val / mY);
```

Comment:
I feel the names for the parameters would read more easily if they followed the same pattern as humidity, i.e. mX + mY * (val / mZ). Thoughts?